app/actions/UserActions.js

```diff
@@ -63,11 +63,13 @@ export function login(
     });
 }
 
-export function logout() {
+export function logout(redirect: boolean = true) {
   return (dispatch: Dispatch<*>) => {
     removeToken();
     dispatch({ type: User.LOGOUT });
-    dispatch(replace('/'));
+    if (redirect) {
```

Comment:
wouldn't it be just as good to just dispatch this after dispatching `logout`? not a fan of boolean arguments :P

Reply:
`push` that is

Reply:
:+1: